modules/quadbin/snowflake/sql/QUADBIN_BOUNDARY.sql

```diff
@@ -0,0 +1,70 @@
+----------------------------
+-- Copyright (C) 2022 CARTO
+----------------------------
+CREATE
+OR REPLACE FUNCTION QUADBIN_BOUNDARY(quadbin INT) 
+RETURNS GEOGRAPHY 
+AS $$ 
+CASE
+    WHEN quadbin IS NULL THEN NULL -- Deal with level 0 boundary issue.
+    WHEN quadbin = 5192650370358181887 THEN TRY_TO_GEOGRAPHY(
```

Comment:
We could use TO_GEOGRAPHY here in this cases with fixed input, so we make sure our geometry literals are OK.